### ISSUE TITLE
Error with Ledger Nano S Plus - not able to sign for Polkadot.js web extension

### ISSUE THREAD

**Issue by rgroener** [polkadot-js/extension#1326]
* **I'm submitting a ...**

  - [ ] Bug report
  - [ ] Feature request
  - [X] Support request
  - [ ] Other

* **What is the current behavior and expected behavior?**
Signing transactions with Ledger Nano S not  possible anymore.
Worked fine until last week. 
Error-Message when trying to sign via the polkadot.js extension:

Ledger error: Failed to execute 'requestDevice' on 'USB': Must be handling a user gesture to show a permission request.

* **Please tell us about your environment:***

 OS: Linux (Arch)
Browser: Chrome (Version 122.0.6261.94 (Official Build) (64-bit))
App and Firmware on Ledger Nano S are up to date. (Updated after the error occurred, therefore the new app/fw version should not be the problem...)

Interacting with Ledger-Live works normal. To me, it seems to be a problem with the polkadot.js extension.

On a different device, the same error is showing up.

Thanks in advance...

**Comment by Tbaut** [comment]
Thanks a lot for the report @rgroener, I'll try to reproduce shortly. Can you tell whether this is working fine with another extension such as Talisman?

**Comment by rgroener** [comment]
@Tbaut, thanks for your fast response.
I found a solution in the meantime. I had to "open the extension in new window" and was then able to connect and sign the transaction.

I have no idea, what triggered this behavior...

I found the workaround on  stackexchange.com 
 [https://substrate.stackexchange.com/questions/2028/issues-with-ledger-nano-s-plus-not-able-to-sign-for-polkadot-js-web-extension](url)

Thanks for your support
